src/executors/default.yml

```diff
@@ -0,0 +1,8 @@
+description: "The official CircleCI PHP Docker image."
+parameters:
+  tag:
+    description: "The `circleci/php` Docker image version tag."
```

Comment:
Could we change this to our new beta cimg docker image while we are here?

> cimg/php

https://hub.docker.com/r/cimg/php

Reply:
Done in https://github.com/CircleCI-Public/php-orb/pull/11/commits/190b3f58ceb8b6988b76bd53100b9f891ab6f5f0

Reply:
Changes in this PR would overwrite these changes.

Reply:
merged from upstream